src/Bundle/Resources/config/services/integrations/translation.xml

```diff
@@ -17,9 +17,9 @@
 
         <service id="sylius.translation_locale_provider.immutable" class="Sylius\Component\Resource\Translation\Provider\ImmutableTranslationLocaleProvider">
             <argument type="collection">
-                <argument>%locale%</argument>
+                <argument>%sylius.resource.translation.enabled_locales%</argument>
```

Comment:
`%sylius.resource.translation.enabled_locales%` should replace the whole collection.

Reply:
I don't understand what you mean. Do you have an example?

Reply:
From this
```xml
        <service id="sylius.translation_locale_provider.immutable" class="Sylius\Component\Resource\Translation\Provider\ImmutableTranslationLocaleProvider">
            <argument type="collection">
                <argument>%sylius.resource.translation.enabled_locales%</argument>
            </argument>
            <argument>%sylius.resource.translation.default_locale%</argument>
        </service>
```
to this
```xml
        <service id="sylius.translation_locale_provider.immutable" class="Sylius\Component\Resource\Translation\Provider\ImmutableTranslationLocaleProvider">
            <argument>%sylius.resource.translation.enabled_locales%</argument>
            <argument>%sylius.resource.translation.default_locale%</argument>
        </service>
```

`%sylius.resource.translation.enabled_locales%` is already an array, so the current implementation will send an array like this `[['en']]` to the `ImmutableTranslationLocaleProvider` instead of the expected `['en']`

Reply:
done